domains/asnct.json

```diff
@@ -1,11 +1,13 @@
 {
-  "description": "Redirect to my telegram channel",
+  "description": "Redirect to my telegram channel & mail",
   "owner": {
     "username": "asnct",
     "email": "asnct@proton.me"
   },
   "record": {
     "URL": "https://asnct_bio.t.me",
-    "TXT": "mailru-domain: wDSaV835X4LZyKsP"
+    "TXT": "mailru-domain: wDSaV835X4LZyKsP",
+    "TXT": "v=spf1 redirect=_spf.mail.ru",
```

Comment:
When you have multiple values for the same record, use an array.

```json

{
   "TXT": ["mailru-domain: wDSaV835X4LZyKsP", "v=spf1 redirect=_spf.mail.ru"]
}

Reply:
Done